pkg/master/controller.go

```diff
@@ -83,6 +86,11 @@ func (c *Controller) Start() {
 	repairClusterIPs := servicecontroller.NewRepair(c.ServiceClusterIPInterval, c.ServiceRegistry, c.ServiceClusterIPRange, c.ServiceClusterIPRegistry)
 	repairNodePorts := portallocatorcontroller.NewRepair(c.ServiceNodePortInterval, c.ServiceRegistry, c.ServiceNodePortRange, c.ServiceNodePortRegistry)
 
+	// Initialize the master leases
+	if c.MasterLeases != nil {
```

Comment:
Why is this done here, and not the responsibility of the person who sets up the controller?